lib/client/client.js

```diff
@@ -110,12 +110,17 @@ function Client (options) {
   EventEmitter.call(this, options)
 
   var self = this
-  var _url
-  if (options.url) { _url = url.parse(options.url) }
-  this.host = _url ? _url.hostname : undefined
-  this.port = _url ? _url.port : false
-  this.secure = _url ? _url.secure : false
-  this.url = _url
+  this.servers = []
+  if (options.url) {
+    var [, protocol, hosts, path] = options.url.match(/^([a-zA-Z]+:\/\/)([^/]*)(.*)/)
+
+    hosts.split(',').forEach((host) => {
+      this.servers.push(url.parse(`${protocol}${host}${path}`))
+    })
+  }
```

Comment:
This code can be simplified to `this.servers = Array.isArray(opts.url) ? opts.url : [opts.url]` when we do not accept comma separated strings.